src/wp-includes/option.php

```diff
@@ -247,6 +247,105 @@ function get_option( $option, $default_value = false ) {
 	return apply_filters( "option_{$option}", maybe_unserialize( $value ), $option );
 }
 
+/**
+ *
```

Comment:
Function Summary and Description is missing.

Reply:
Fixed.